plugins/CoreHome/stylesheets/layout.less

```
@@ -190,7 +190,7 @@ nav {
       padding-bottom: 20px;
 
       .menu-icon {
-        padding-right: 13px;
+        padding-right: 9.5px;
```

Guess this got reverted due to the merge:
```suggestion
        padding-right: 13px;
```